staubli_rx160_support/urdf/rx160l_macro.xacro

```diff
@@ -68,7 +68,7 @@
 			<visual>
 				<origin xyz="0 0 0" rpy="0 0 0"/>
 				<geometry>
-				    <mesh filename="package://staubli_rx160_support/meshes/rx160l/visual/link_4.stl"/>
+					<mesh filename="package://staubli_rx160_support/meshes/rx160l/visual/link_4.stl"/>
```

Comment:
All xacros & urdfs in this pkg use tabs for indentation, except for this line (it uses three spaces). That was introduced in #10 and this change fixes that.

Does that address your concern?